app/src/test/java/com/github/multimatum_team/multimatum/AddDeadlineTest.kt

```diff
@@ -90,13 +95,22 @@ class AddDeadlineTest {
         )
     }
 
+    @Test
+    fun `parsing validation pop-up`(){
```

Comment:
You could just add the line to click on the ok button to increase a little the coverage and to be sure that the dialog disappear and add another test to test the cancel option but this is minimal

Reply:
I did that